assets/php/class-wgpb-block-product-best-sellers.php

```diff
@@ -4,7 +4,7 @@
  * NOTE: DO NOT edit this file in WooCommerce core, this is generated from woocommerce-gutenberg-products-block.
  *
  * @package WooCommerce\Blocks
- * @version 2.1.0
+ * @version 2.2.0
```

Comment:
Do we actually need to track file versions like this?

Reply:
I was just following what's already done in Woo core, what do you think?

Reply:
Personally I'd remove them. When refactoring core classes I've had been removing them before I switched teams.

Reply:
OK, sure – will remove these, especially since the don't make sense once copied into WC core 👍